### ISSUE TITLE
Transfer Learning of the model for PASCAL VOC dataset

### ISSUE THREAD

**Issue by sreesindhu-sabbineni** [Attila94/refinenet-keras#5]
Thank you for responding to my previous query in the other issue. I want to apply transfer learning to the model for PASCAL VOC dataset. I have used the weights file you have provided for CityScapes dataset as the initial weights for the model. I have only changed the number of classes which is only used at the last layer. But when I tried to see some initial predictions, all I see are a bunch 0's and 1's (basically some noise). I have also changed the activation function in the final layer to 'Sigmoid'. I thought I should get some basic segmentation results as the model is pretrained with CityScapes dataset but it didn't happen. Do you think it is because of the usage of the 'Sigmoid' activation function? Do we have to use only 'Softmax' activation for segmentation tasks? For PASCAL VOC, each image contains multiple objects and I see CityScapes also contains multiple objects.

**Comment by Attila94** [comment]
Since a pixel can only belong to a single class Softmax is the way to go. Also since you changed the number of classes in the last layer, you basically threw away its weights and reinitialised it with random weights.
